firebase-ai/src/main/kotlin/com/google/firebase/ai/common/APIController.kt

```diff
@@ -97,7 +97,7 @@ internal constructor(
   private val requestOptions: RequestOptions,
   httpEngine: HttpClientEngine,
   private val apiClient: String,
-  private val firebaseApp: FirebaseApp,
+  internal val firebaseApp: FirebaseApp,
```

Comment:
why the change?

Reply:
to access it from the LiveGenerativeModel.kt

Reply:
Done